mlir/test/lib/Dialect/Vector/TestVectorTransforms.cpp

```diff
@@ -874,6 +874,38 @@ struct TestVectorLinearize final
       return signalPassFailure();
   }
 };
+
+struct TestEliminateVectorMasks
+    : public PassWrapper<TestEliminateVectorMasks,
+                         OperationPass<func::FuncOp>> {
+  MLIR_DEFINE_EXPLICIT_INTERNAL_INLINE_TYPE_ID(TestEliminateVectorMasks)
+
+  TestEliminateVectorMasks() = default;
+  TestEliminateVectorMasks(const TestEliminateVectorMasks &pass)
+      : PassWrapper(pass) {}
+
+  Option<unsigned> vscaleMin{
+      *this, "vscale-min",
+      llvm::cl::desc(
+          "Minimum value `vector.vscale` can possibly be at runtime."),
```

Comment:
nit:
```suggestion
          "Minimum value of `vector.vscale` at runtime."),
```

Reply:
This has a different meaning to me. 
i.e. the minimum value cross all possible runtimes vs the minimum value of vscale for a specific runtime.

Reply:
not sure I follow, but I don't think it necessary to mention runtime anyway, "Minimum value of `vscale.vscale`" is sufficient.

Reply:
I've shortened it to "Minimum possible value of vscale." (which is used elsewhere too)